packages/docs/base/typography.md

```diff
@@ -12,14 +12,14 @@ description: A set of pre-defined text styles for headers, body copy, & links de
 <p>Odyssey offers separate font stacks for UI, copy, and code:
 
 ```scss
-$body-font-family: 'Public Sans', 'Noto Sans', 'Whyte', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
+$body-font-family: 'Public Sans', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
 
 $mono-font-family: 'Inconsolata', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
 ```
 
 </p>
 
-UI and copy both utilize our `$body-font-family`. This stack prioritizes Public Sans. This is followed by Noto Sans to support internationalization, then Whyte for Okta-branded pages. Finally, we utilize a system font-stack to enable device-standard typography when our preferred families aren't available.
+UI and copy both utilize our `$body-font-family`. This stack prioritizes Public Sans. We fall back to a system font-stack to enable device-standard typography when our preferred families aren't available.
```

Comment:
this is interesting - we wouldn't fall back to other sans-serif fonts prior to system fonts? I'm not sure if any system fonts use serifs but it does seem unexpected

Reply:
Once upon a time, ODS was built specifically for system fonts as a way to represent Okta's brand of neutrality. Since then, we've been shifting our stack to ensure a branded font comes first, with system as the fallback.

@joejezowski-okta may have more thoughts here on an alternate solution